src/Common/Core/Impl/Telemetry/TelemetryServiceBase.cs

```diff
@@ -32,13 +32,13 @@ public abstract class TelemetryServiceBase : ITelemetryService, IDisposable {
         /// </summary>
         public bool IsEnabled {
             get {
-                return this.TelemetryRecorder.IsEnabled;
+                return this.TelemetryRecorder?.IsEnabled == true;
```

Comment:
[https://github.com/Microsoft/RTVS/issues/2174](https://github.com/Microsoft/RTVS/issues/2174)

---
In reply to: [74838090](https://github.com/Microsoft/RTVS/pull/2173#discussion_r74838090) [](ancestors = 74838090,74833030,74832919,74832126,74828037)